src/map/clif.c

```diff
@@ -9851,21 +9851,37 @@ static int clif_guess_PacketVer(int fd, int get_previous, int *error)
 		err = 0;\
 		if( error )\
 			*error = 0;\
+		if( diskey ) ShowWarning("Client use no encryption while server do, please fix me\n"); \
```

Comment:
Client is not sending encrypted packets but the server is setup to accept encrypted packets. Please disable PACKET_OBFUSCATION.